src/http.rs

```diff
@@ -256,15 +256,21 @@ pub async fn run_server(
 		let counter_lock = counter_status.lock().unwrap();
 		status(&cfg, *counter_lock, db.clone())
 	});
+	let cors = warp::cors()
+		.allow_any_origin()
+		.allow_header("content-type")
+		.allow_methods(vec!["GET", "POST", "DELETE"]);
```

Comment:
We do not allow any POST or DELETE at the moment, but IMO we should not allow any origin for them. This legacy code, if remain unchanged, can be a security vulnerability later.

Reply:
Never understood CORS implications completely :) This is only for user agents to allow requests to LC domain after OPTIONS call. I agree is better to have narrowest settings possible (like CORS setting per endpoint), but I don't understand what are vulnerabilities :)